pages/pix-pro/locale-home.vue

```diff
@@ -15,22 +15,34 @@
 <script>
 import { documentFetcher, DOCUMENTS } from '~/services/document-fetcher'
 
+const EN_HOME_DOCUMENT_UID = 'discover-pix-pro'
+const FR_HOME_DOCUMENT_UID = 'decouvrir-pix-pro'
+
+const HOME_DOCUMENT_UID_BY_LOCALE = {
```

Comment:
Il me semble qu'un nommage de ces différentes variables avec un même préfixe améliorerait la lecture et la cohérence :

```suggestion
const HOME_DOCUMENT_UID_EN = 'discover-pix-pro'
const HOME_DOCUMENT_UID_FR = 'decouvrir-pix-pro'

const HOME_DOCUMENT_UID_BY_LOCALE = {
```